### PR TITLE
[ACR-1319]Add external storage feature

### PR DESCRIPTION
**Pull request by prjung**
Change Description
[ACR-1319]Add external storage feature

Bugs Fixed
N/A

API Changes
ACR-1319

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/567

File: docs/application/dotnet/getting-started/app-filtering.md [hunk 1]
```diff
@@ -238,6 +238,7 @@ The following tables show the available requirements for an application package.
 | `http://tizen.org/feature/speech.control` | Specify this key, if the application requires the voice control feature. | 2.4   |
 | `http://tizen.org/feature/speech.recognition` | Specify this key, if the application requires the speech recognition (STT) feature. | 2.2.1 |
 | `http://tizen.org/feature/speech.synthesis` | Specify this key, if the application requires the speech synthesis (text to speech, TTS) feature. | 2.2.1 |
+| `http://tizen.org/feature/storage.external` | Specify this key, if the application requires the external storage feature. | 5.5   |
```
Comment (divyk) [inline docs/application/dotnet/getting-started/app-filtering.md]: Please make this small change:

Specify this key, if the application requires an external storage feature.

File: docs/application/dotnet/getting-started/app-filtering.md [hunk 2]
```diff
@@ -394,6 +395,7 @@ The following tables show the available requirements for an application package.
...
```
Comment (divyk) [inline docs/application/dotnet/getting-started/app-filtering.md]: Please make this small change:

Specify this key, if the application requires an external storage feature.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 3]
```diff
@@ -313,6 +313,7 @@ package:
 | `http://tizen.org/feature/speech.control` | Specify this key, if the application requires the voice control feature. | 2.4   |
 | `http://tizen.org/feature/speech.recognition` | Specify this key, if the application requires the speech recognition (STT) feature. | 2.2.1 |
 | `http://tizen.org/feature/speech.synthesis` | Specify this key, if the application requires the speech synthesis (text to speech, TTS) feature. | 2.2.1 |
+| `http://tizen.org/feature/storage.external` | Specify this key, if the application requires the external storage feature. | 5.5   |
```
Comment (divyk) [inline docs/application/native/tutorials/details/app-filtering.md]: Please make this small change:

Specify this key, if the application requires an external storage feature.

File: docs/application/native/tutorials/details/app-filtering.md [hunk 4]
```diff
@@ -477,6 +478,7 @@ Reference](../../api/overview.md).
...
```
Comment (divyk) [inline docs/application/native/tutorials/details/app-filtering.md]: Please make this small change:

Specify this key, if the application requires an external storage feature.

File: docs/application/native/guides/device/system.md [hunk 5]
```diff
@@ -631,6 +632,15 @@ The following table lists the speech feature keys:
...
 | `http://tizen.org/feature/speech.synthesis` | `bool` | The platform returns `true` for this key, if the device supports speech synthesis (TTS). |
 
+The following table lists the external storage feature keys:
+
+<a name="storage"></a>
+**Table: External storage feature keys**
+
+| Key                                 | Type   | Description                              |
+|-------------------------------------|--------|------------------------------------------|
+| `http://tizen.org/feature/storage.external` | `bool` | The platform returns `true` for this key, if the device supports the external storage. |
```
Comment (divyk) [inline docs/application/native/guides/device/system.md]: Please make this small change:

The platform returns `true` for this key, if the device supports external storage.

File: docs/application/dotnet/getting-started/app-filtering.md (hunk 1 above)
Comment (mijin85cho) [reply docs/application/dotnet/getting-started/app-filtering.md]: @divyk How about getting the consistency with other sentences?

File: docs/application/dotnet/getting-started/app-filtering.md (hunk 2 above)
Comment (mijin85cho) [reply docs/application/dotnet/getting-started/app-filtering.md]: @divyk How about getting the consistency with other sentences?

File: docs/application/native/tutorials/details/app-filtering.md (hunk 3 above)
Comment (mijin85cho) [reply docs/application/native/tutorials/details/app-filtering.md]: @divyk How about getting the consistency with other sentences?

File: docs/application/dotnet/getting-started/app-filtering.md (hunk 1 above)
Comment (divyk) [reply docs/application/dotnet/getting-started/app-filtering.md]: Ok, I agree, we can retain the original sentence. Thanks.

File: docs/application/dotnet/getting-started/app-filtering.md (hunk 2 above)
Comment (divyk) [reply docs/application/dotnet/getting-started/app-filtering.md]: Ok, I agree, we can retain the original sentence. Thanks.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 3 above)
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree, we can retain the original sentence. Thanks.

File: docs/application/native/tutorials/details/app-filtering.md (hunk 4 above)
Comment (divyk) [reply docs/application/native/tutorials/details/app-filtering.md]: Ok, I agree, we can retain the original sentence. Thanks.

File: docs/application/native/guides/device/system.md (hunk 5 above)
Comment (prjung) [reply docs/application/native/guides/device/system.md]: Done.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/567

**Comment by mijin85cho** [comment]
@prjung According to [ACR-1319], it looks like web is also target of this update. Don't you need to update docs/application/web/tutorials/app-filtering.md, too?

**Pull request closed without merging**